### PR TITLE
CNV- 31344: FIPS and TLS known issue (4.13.3)

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.13.3

Issue: [CNV-31344](https://issues.redhat.com//browse/CNV-31344)

Link to docs preview: [Known issues](https://62760--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-known-issues)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -201,6 +201,13 @@ link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_mana
 [id="virt-4-13-known-issues"]
 == Known issues
 
+// CNV-31344 FIPS and TLS. RN text is from BZ2157951
+* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (RFC 7627) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected.
++
+Legacy clients that do not support EMS or TLS 1.3 now cannot connect to FIPS servers running on RHEL 9. Similarly, RHEL 9 clients in FIPS mode cannot connect to servers that only support TLS 1.2 without EMS. This in practice means that these clients cannot connect to servers on RHEL 6, RHEL 7 and non-RHEL legacy operating systems. This is because the legacy 1.0.x versions of OpenSSL do not support EMS or TLS 1.3. For more information, see link:https://access.redhat.com/solutions/7018256[TLS Extension "Extended Master Secret" enforced with Red Hat Enterprise Linux 9.2].
++
+The workaround is to configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
```
Comment (tiraboschi) [inline virt/virt-4-13-release-notes.adoc:209]: this is not consistent with the intro.

Comment (apinnick) [reply virt/virt-4-13-release-notes.adoc:209]: You're right. So which is correct?

**Review by kelbrown20 — COMMENTED** [review]
Great work on this!
/lgtm

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**